app/models/energy/ScheduleFixedInterval.py

```diff
@@ -0,0 +1,63 @@
+"""Schedule Fixed Interval Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError, UrlStr, ConstrainedStr
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+import datetime
+from app.models.energy.ScheduleBase import ScheduleContinuous, ScheduleDiscrete, ScheduleNumericType, ScheduleUnitType, Date, Time
+
+
+class ScheduleFixedInterval(BaseModel):
+    """Used to specify a start date and a list of values for a period of analysis."""
+
+    type: Enum('ScheduleFixedInterval', {'type': 'ScheduleFixedInterval'})
+
+    name: str = Schema(
+        ...,
+        regex=r'^[\s.A-Za-z0-9_-]*$',
+    )
+
+    start_date: Date
+
+    is_leap_year: bool = Schema(
```

Comment:
This should also go to date otherwise you will have issues in supporting leap_year. Right now the Date is set to 2016 which I assume is not a leap year.